install/third-party/cloudfoundry/install.yml

```diff
@@ -0,0 +1,14 @@
+id: third-party-cloudfoundry
+name: CloudFoundry
+title: CloudFoundry
+description: |
+  Create, deploy, and manage microservice applications in Kubernetes containers.
+
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/vmware-tanzu-monitoring-integration
```

Comment:
Hey @vchandana25 ! Just wanted to double check this is the correct URL - We have the same link for `vmware-tanzu` in a different PR 😄